### ISSUE TITLE
POST /webauthn/register 422 at webauthn.js:159

### ISSUE THREAD

**Issue by fidelisepeter** [Laragear/WebAuthn#19]
### HELP

Please i am trying to use webauthn in my app i had to upgrade to laravel 9 to user Laragear/WebAuthn because i was unable to set up Larapass and i can't create any issue there.

Wel Successfull Set WebAuthn but having Issue registering Device...
I get 422 Error Code

Actually need Help is not a bug

**Comment by Bubka** [comment]
Hi,

Until PR #20 is merged a workaround is to duplicate the `webauthn` config array like this:

```php
return [
    'relying_party' => [
        'name' => env('WEBAUTHN_NAME', config('app.name')),
        'id'   => env('WEBAUTHN_ID'),
    ],

    'relaying_party' => [
        'id'   => env('WEBAUTHN_ID'),
    ],
]
```

This way whatever the config array the package calls, there is always some values returned.

**Comment by fidelisepeter** [comment]
@Bubka 
Thanks
I tried that I still get same error
I posted on [Stackoverflow](https://stackoverflow.com/questions/74174289/)
I wish this will be solved

**Comment by DarkGhostHunter** [comment]
Fixed by #20
